jakartaee9/src/ejb-jar_4_0.xsds

```diff
@@ -2854,9 +2853,9 @@
               superclass around-timeout methods.
 	    - an optional declaration of the bean's 
 	      environment entries
-	    - an optional declaration of the bean's EJB references
+	    - an optional declaration of the bean's references
 	    - an optional declaration of the bean's local 
-	      EJB references
+	      references
```

Comment:
Should be "the bean's local enterprise bean references".  This is about a bean having references to other beans